index.html

```diff
@@ -160,5 +160,22 @@ <h3 class="css">CSS</h3>
             </ul>
             <img src="img/Separator Bottom.svg" alt="" class="skills-frameworks-divider">
     </section>
+    <section class="fotter-contact-form-container">
+      <section class="fotter-contact-form-text">
+        <h2 class="fotter-contact-form-heading">Contact me</h2>
+        <p class="fotter-contact-form-subheading">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+          I’d love to help with it!</p>
+        <section class="form-conatiner">
+          <form action="https://formspree.io/f/{form_id}" method="post" class="form">
+            <input type="text" name="name" required="required" maxlength="30" placeholder="Lorem Ipsum" class="input name">
+            <input type="email" name="email" required="required" placeholder="loremipsum@gmail.com" class="input email">
+            <textarea name="textarea" class="input textarea" required="required" placeholder="Write your message here.." cols="30" rows="7"></textarea>
+            <input type="submit" value="Get in touch" class="input form-btn">
+          </form>
```

Comment:
- [ ] Kindly make sure that the  **form is submitted successfully** with valid input values.When I submit the form the I got the following error , Kindly Fix it when you integrate with [formspree service](https://formspree.io/html), you didin't create a hash id for the form So Kindly Fix it.
![form](https://github.com/ali-techworld/Portfolio-Setup/assets/93598189/02ec4e96-dac4-47b6-a1ec-df975ccdcba8)